SPEC/BITSWAP.md

```diff
@@ -4,13 +4,37 @@
 * [bitswap.unwant](#bitswapunwant)
 * [bitswap.stat](#bitswapstat)
 
-#### `bitswap.wantlist`
+#### `bitswap.unwant`
 
-(not spec'ed yet)
+> Removes a given block from your wantlist
 
-#### `bitswap.unwant`
+##### `Go` **WIP**
+
+##### `JavaScript` - ipfs.bitswap.unwant(cid)
+
+`cid` is a [cid][cid] which can be passed as:
+
+- CID, a CID instance
+- String, the base58 encoded version of the multihash
+
+##### `Go` **WIP**
+
+### `bitswap.wantlist`
+
+> Returns the wantlist, optionally limited by peerID
```

Comment:
It would be helpful to clarify if "the wantlist" when not limited by peer ID is a list that everybody shares or if it is just yours.

Also, would "filtered" be a more accurate description than "limited"?